src/components/breadcrumbs/examples/breadcrumbs-buttons.tsx

```diff
@@ -0,0 +1,66 @@
+import { Component, h, State } from '@stencil/core';
+import {
+    BreadcrumbsItem,
+    LimelBreadcrumbsCustomEvent,
+} from '@limetech/lime-elements';
+
+/**
+ * Items as buttons
+ * The Breadcrumbs can also be used to navigate between different
+ * steps of a process, such as steps of a form or survey, or
+ * moving through steps of a wizard.
+ *
+ * In this case, you will not provide any `link`s and instead will
+ * handle the clicks. When no links are provided, the component
+ * will automatically generate a list of `buttons`s.
+ *
+ * Keep in mind that the last item will not be rendered as an
+ * HTML button and and therefore won't be clickable.
+ */
+@Component({
+    tag: 'limel-example-breadcrumbs-buttons',
+    shadow: true,
+})
+export class BreadcrumbsExampleButtons {
```

Comment:
We usually add `Example` as postfix. Check it in other component examples
```suggestion
export class BreadcrumbsButtonsExample {
```